### ISSUE TITLE
Custom Paintings Missing Textures

### ISSUE THREAD

**Issue by LordRegent303** [Nuchaz/BiblioCraft#498]
I added my custom paintings resource pack and every one of them has missing texture (the magenta/black checkerboard). Other paintings work fine. I'm using forge-1.12-14.21.1.2443 with some mods: CXLibrary, Iron Chests, More Furnaces, WorldEdit, WorldEdit CUI, Chisel and Bits, OptiFine, JEI, Xaero's Minimap and ToroHealth Damage Indicators. If you need more information - I'll provide it!

**Comment by AaronBPBC** [comment]
Are all of the paintings lowercase with no other characters or spaces? That was my problem. (Ex: cowlover)

**Comment by LordRegent303** [comment]
I only have lowercase and uppercase letters in names without any spaces or characters. Some have numbers though (Ex: Banner1)

**Comment by AaronBPBC** [comment]
Yeah, there is your issue. You can't have any uppercase letters. I'm not sure if you can have numbers, but just to be safe I just typed out the numbers. If only using lowercase doesn't work, I don't know what else.

**Comment by LordRegent303** [comment]
Thank you! It worked.

**Comment by DaSausage** [comment]
Very much useful, thanks

**Comment by D3Gamer** [comment]
This Helped Me!

**Comment by LithTheKitten** [comment]
what about the numbers
